neqo-transport/src/connection/mod.rs

```diff
@@ -1101,10 +1101,18 @@ impl Connection {
     #[must_use = "Output of the process function must be handled"]
     pub fn process(&mut self, dgram: Option<&Datagram>, now: Instant) -> Output {
         if let Some(d) = dgram {
+            #[cfg(feature = "build-fuzzing-corpus")]
+            neqo_common::write_item_to_fuzzing_corpus("packet", d);
```

Comment:
I think that you should avoid the use of *input* datagrams unless you can show that they are valid.  I think that it is enough to check the output datagrams.

Reply:
Should I do the same with frames? At the moment, I only save _inbound_ frames, not generated ones.

Reply:
Merging this now. We can always change things.